packages/dotcom-ui-header/browser.js

```diff
@@ -16,7 +17,29 @@ export const init = (headerOptions = {}) => {
     '.o-header [data-n-topic-search], .o-header__drawer [data-n-topic-search]'
   )
   topicSearchElements.forEach((element) => {
-    new TopicSearch(element, headerOptions)
+    const oheaderContainer = element.parentElement
+    const oheaderContainerParent = oheaderContainer?.parentElement
+    const isDrawer = oheaderContainerParent?.parentElement?.getAttribute('data-o-header-drawer') === 'true'
+
+    const form = oheaderContainer?.querySelector('form')
+    const input = form?.querySelector('input')
+    const typeaheadContainer = document.createElement('div')
+    typeaheadContainer.id = `suggestions-${input?.id}`
+    typeaheadContainer.className = 'typeahead__main-container'
+    typeaheadContainer.role = 'listbox'
+
+    if (!form || !input) return
+
+    if (isDrawer) {
+      const mobileDrawer = oheaderContainerParent?.parentElement
+      element.appendChild(typeaheadContainer)
+
+      render(<TypeAhead container={mobileDrawer} inputId={input.id} />, typeaheadContainer)
+    } else {
+      form.after(typeaheadContainer)
```

Comment:
could be better check if form exists here?

Reply:
Replied [here](https://github.com/Financial-Times/dotcom-page-kit/pull/1052/files#r1736462042).